src/tags/if_block.rs

```diff
@@ -25,6 +26,19 @@ struct Conditional {
     if_false: Option<Template>,
 }
 
+fn contains_check(a: Value, b: Value) -> Result<bool> {
+    let a = match a.as_str() {
+        Some(s) => s,
+        None => return Error::renderer("Left-hand side of contains operator must be a string"),
```

Comment:
(this is non-essential feedback, feel free to ignore)

Personally, I prefer
```rust
let a = a.as_str().ok_or("Left-hand side of contains operator must be a string")?;
```
or more like yours
```rust
let a = a.as_str().ok_or_else(|| Error::renderer("Left-hand side of contains operator must be a string"))?;
```